en/user_guide/node.rst

```diff
@@ -0,0 +1,41 @@
+Node
+====
+
+Languages
+---------
+
+A node has several languages defined during the configuration of Back Office.
+Clicked on the language tab to change the node language. The first time it will create your node in this language.
```

Comment:
"The node language being edited can be changed by clicking on the language tab. If this node has no data for this language, it will be created." (maybe not accurate, to rephrase)